testing/functional/tests/cypress/fixtures/instituteManageLocationData.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Please use the word Institution instead of Institute as it is used all over the system.

Reply:
Updated.